cmd/brimcap/root/command.go

```diff
@@ -39,26 +39,21 @@ through using brimcap search.
 	New: New,
 }
 
+var LogJSON bool
+
 type Command struct {
 	charm.Command
 	cli cli.Flags
-	// Child is set by the select Child command.
-	Child ChildCmd
-	JSON  bool
 }
 
 func New(parent charm.Command, f *flag.FlagSet) (charm.Command, error) {
 	c := &Command{}
 	c.cli.SetFlags(f)
 	isterm := term.IsTerminal(int(os.Stdout.Fd()))
-	f.BoolVar(&c.JSON, "json", !isterm, "encode stderr in json")
+	f.BoolVar(&LogJSON, "json", !isterm, "encode stderr in json")
```

Comment:
Nit: Moving `LogJSON` to package scope feels unfortunate, but I don't have a better idea to offer.

Reply:
Agreed.